### ISSUE TITLE
Gray & skewed output and lack of orientation support

### ISSUE THREAD

**Issue by karolkulesza** [alexp2ad/record-ios-simulator#1]
Hi @alexp2ad,

Cool stuff!

Attached below you can find my output, after testing against iOS 12.1 iPad Pro simulator (in horizontal orientation)

[Simulator 2019-01-11 at 22.06.48.mp4.zip](https://github.com/alexp2ad/record-ios-simulator/files/2751016/Simulator.2019-01-11.at.22.06.48.mp4.zip)

macOS: v10.13.6
Xcode: v10.1

I assume those issues are shortcomings of `xcrun simctl`?

**Comment by alexjsp** [comment]
Hmm, the grey and skewed output I've never seen before and can't replicate. (But ultimately is probably `simctl`'s fault, since it does all of the actual recording 🤷‍♂️ )

The lack of correct orientation is definitely an expected weakness in `simctl`, but it's fairly easy to fix after the recording with Quicktime or Handbrake.

**Comment by karolkulesza** [comment]
Would you mind sharing your environment setup? (Xcode version, macOS version, which iOS simulator precisely, etc).

**Comment by alexjsp** [comment]
Sure:

macOS: 10.14.2
Xcode: 10.1 (10B61)
Xcode Command Line Tools: 10.1 (10B61)

Re: simulator I've tried it with a bunch, including iPhone X, current iPad Pros, Apple TV etc.  Probably only tested against recent iOS / tvOS versions, but I don't _think_ that matters.

Do you always see that weird skewed output at the start?

**Comment by karolkulesza** [comment]
Yep, unfortunately it's always there (both grayed and skewed). Since I'm using the same Xcode version, I suspect it might be a matter of different macOS version (I'm still on v10.13).

**Comment by alexjsp** [comment]
Possible, but the recording functionality has been around for a couple years in Xcode so it's weird it wouldn't work right on High Sierra.

Any software installed that might be interfering with the GPU / simulator or something weird? (Or are you on a hackintosh? 🤔 )

**Comment by karolkulesza** [comment]
No hackintosh, but I've installed a bunch of different GPU-related software in the past few years on this machine, so it might be a case. I guess it'll be quicker to setup a fresh macOS installation, though  🤕
